src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
-}
+  const rentCar = 40;
+  let sumRent = 0;
+
+  switch (true) {
+    case (days >= 3 && days <= 6):
+      sumRent = (days * rentCar) - 20;
+      break;
+    case (days >= 7):
+      sumRent = (days * rentCar) - 50;
```

Comment:
Try to avoid magic numbers. All numbers that are unclear from the context should be put into constants with a name that is understandable in terms of content